cmd/exporttool/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/temporalio/cloud-samples-go/export"
+)
+
+func main() {
+	if len(os.Args) != 2 {
```

Comment:
maybe use urfave?  or it's overkill 😄

Reply:
i think its probably a bit overkill for this. the goal here is to provide baseline sample code on how to process an export file